test/jdk/javax/swing/JMenuBar/TestMenuMnemonic.java

```diff
@@ -0,0 +1,127 @@
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 8326458
+ * @key headful
+ * @requires (os.family == "windows")
+ * @modules java.desktop/com.sun.java.swing.plaf.windows
+ * @summary Verifies if menu mnemonics toggle on F10 press in Windows LAF
+ * @run main TestMenuMnemonic
+ */
+
+import java.awt.event.KeyEvent;
+import java.awt.Robot;
+import java.util.concurrent.atomic.AtomicInteger;
+import javax.swing.JFrame;
+import javax.swing.JMenu;
+import javax.swing.JMenuItem;
+import javax.swing.JMenuBar;
+import javax.swing.MenuElement;
+import javax.swing.MenuSelectionManager;
+import javax.swing.SwingUtilities;
+import javax.swing.UIManager;
+import com.sun.java.swing.plaf.windows.WindowsLookAndFeel;
+
+public class TestMenuMnemonic {
+
+    private static JFrame frame;
+    private static JMenuBar menuBar;
+    private static JMenu fileMenu;
+
+    public static void main(String[] args) throws Exception {
+        UIManager.setLookAndFeel("com.sun.java.swing.plaf.windows.WindowsLookAndFeel");
+        int expectedMnemonicShowHideCount = 5;
+        Robot robot = new Robot();
+        robot.setAutoDelay(100);
+        AtomicInteger mnemonicHideCount = new AtomicInteger(0);
+        AtomicInteger mnemonicShowCount = new AtomicInteger(0);
+
+        try {
+            SwingUtilities.invokeAndWait(TestMenuMnemonic::createAndShowUI);
+            robot.waitForIdle();
+            robot.delay(1000);
+
+            for (int i = 0; i < 10; i++) {
+                robot.keyPress(KeyEvent.VK_F10);
+                robot.waitForIdle();
+                robot.delay(50);
+                robot.keyRelease(KeyEvent.VK_F10);
+                robot.waitForIdle();
+                robot.delay(50);
+                SwingUtilities.invokeAndWait(() -> {
+                    MenuSelectionManager msm =
+                            MenuSelectionManager.defaultManager();
+                    MenuElement[] selectedPath = msm.getSelectedPath();
+                    if (WindowsLookAndFeel.isMnemonicHidden()) {
+                        mnemonicHideCount.getAndIncrement();
+                        // check if selection is cleared when mnemonics are hidden
+                        if (selectedPath.length != 0) {
+                            throw new RuntimeException("Menubar is active after" +
+                                    " mnemonics are hidden");
+                        }
+                    } else {
+                        mnemonicShowCount.getAndIncrement();
+                        if (selectedPath.length != 2 &&
+                                (selectedPath[0] != menuBar || selectedPath[1] != fileMenu)) {
```

Comment:
```suggestion
                        if (selectedPath.length != 2
                            && (selectedPath[0] != menuBar || selectedPath[1] != fileMenu)) {
```
In all the other places you wrap before the operator; in fact it's what Java Coding Style recommends doing. And this style makes it clear that it's a continuation line, if it starts with an operator.